internal/cephfs/volume.go

```diff
@@ -183,19 +191,15 @@ func (vo *volumeOptions) resizeVolume(ctx context.Context, volID volumeID, bytes
 	}
 	// resize subvolume when either it's supported, or when corresponding
 	// clusterID key was not present.
-	if clusterAdditionalInfo[vo.ClusterID].resizeSupported == nil || *clusterAdditionalInfo[vo.ClusterID].resizeSupported {
-		if clusterAdditionalInfo[vo.ClusterID].resizeSupported == nil {
-			clusterAdditionalInfo[vo.ClusterID].resizeSupported = new(bool)
-		}
+	if clusterAdditionalInfo[vo.ClusterID].resizeSupported == resizeUnknown || clusterAdditionalInfo[vo.ClusterID].resizeSupported == resizeSupported {
```

Comment:
it was on a single line, but you may improve readability by putting each statement on a single line